README.md

```diff
@@ -127,25 +157,40 @@ a comma separated list. Each item in the list defines a new route and must also
 ```
 derivative.<item>.enabled=true
 ```
+
 This defines if the `item` service is enabled.
+
 ```
 derivative.<item>.in.stream=queue:islandora-item-connector.index
 ```
+
 This is the input queue for the derivative microservice.
 The part after `queue:` should match your Islandora instance "Actions".
+
 ```
 derivative.<item>.service.url=http://example.org/derivative/convert
 ```
+
 This is the microservice URL to process the request.
+
 ```
 derivative.<item>.concurrent-consumers=1
 derivative.<item>.max-concurrent-consumers=1
 ```
+
 These define the default number of concurrent consumers and maximum number of concurrent
 consumers working off your ActiveMQ instance.
 A value of `-1` means no setting is applied.
 
+
+```
+derivative.<item>.async-consumer=true
+```
+
+This property allows the concurrent consumers to process concurrently; otherwise, the consumers will wait to the previous message has been processed before executing.
+
 For example, with two services defined (houdini and crayfits) my configuration would have
+
 ```
 derivative.systems.installed=houdini,fits
 
```

Comment:
This is a total nit-pick but could you add
```
derivative.houdini.async-consumer=true
```
below as that one has different values for `concurrent-consumers` and `max-concurrent-consumers`.

Otherwise, I'm good with this.

Reply:
Heh... could do: Didn't really want to touch the example stuff where it says "[your] configuration would have [these things]"...